doc/guide/guide-overview.rst

```diff
@@ -4,22 +4,14 @@
 Guide Overview
 ******************
 
-The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview. In addition, the :ref:`apidoc` for each function is located at the end of this guide.
+The goal of this guide is to introduce you to the basic structures and functions that make up QuTiP. This guide is divided up into several sections, each highlighting a specific set of functionalities. In combination with the examples that can be found on the project web page `https://qutip.org/tutorials.html <https://qutip.org/tutorials.html>`_, this guide should provide a more or less complete overview. In addition, the :ref:`apidoc` for each function is located below.
 
 
 .. _overview-org:
 
 Organization
 =============
 
-QuTiP is designed to be a general framework for solving quantum mechanics problems such as systems composed of few-level quantum systems and harmonic oscillators. To this end, QuTiP is built from a large (and ever growing) library of functions and classes; from :func:`qutip.states.basis` to :func:`qutip.wigner`.  The general organization of QuTiP, highlighting the important API available to the user, is shown in the figure below.
-
-
-.. _figure-qutip-org:
-
-.. figure:: figures/qutip_tree.png
-   :align: center
-   :figwidth: 100%
-
-   Tree-diagram of the 468 user accessible functions and classes in QuTiP 4.6. A vector image of the code tree is in :download:`qutip_tree.pdf <doc/qutip_tree.pdf>`.
```

Comment:
Why was the figure removed?

Reply:
I have restored it; I removed it because it didn't convey any information to me other than that there were a large number of objects at two levels. If someone can explain what the figure means to me, I will add an explanatory sentence so that the reader will be able to get something from it.

Reply:
It's intended as a way to get an overview of the structure of the qutip package. The `.png` is really an advertisement for the `.pdf` where one can zoom in nicely and explore the tree.